gammapy/estimators/points/core.py

```diff
@@ -405,7 +423,7 @@ def to_table(self, sed_type=None, format="gadf-sed", formatted=False):
                 table_flat["time_max"] = [time_max.mjd]
 
                 fp = self.slice_by_idx(slices={"time": idx})
-                table = fp.to_table(sed_type=sed_type, format="gadf-sed")
+                table = fp.to_table(sed_type=sed_type)
```

Comment:
The tests pass because once you do a `slice_by_idx` on the time axis, the time axis is removed in the resultant flux point object, and `_guess_format` returns `gadf-sed`. but this is a bit risky and it is better to be explicitly correct